### PR TITLE
completed javascript-modal | Janelle Gatmaitan

### PR DESCRIPTION
**Pull request by JanelleGatmaitan**
image
image

### CODE REVIEW

File: javascript-modal/main.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+var click = true;
+var $openModal = document.querySelector('.openButton');
+var $textBoxContainer = document.querySelector('.textBoxContainer');
+var $overlay = document.querySelector('.overlay');
+var $noButton = document.querySelector('.noButton');
+
+$openModal.addEventListener('click', open);
+
+function open(event) {
+  if (click === true) {
```
Comment (codingcodymiller) [inline javascript-modal/main.js]: It doesn't look like this variable is ever changing in value. You can just delete the variable, and the conditionals referencing it.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work so far! Please see my note

File: javascript-modal/main.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+var $overlay = document.querySelector('.overlay');
+var $noButton = document.querySelector('.noButton');
+
+$openModal.addEventListener('click', open);
+
+function open(event) {
+  if (true) {
+    $textBoxContainer.className = 'textBoxContainer';
+    $overlay.className = 'overlay';
+  }
```
Comment (thebearingedge) [inline javascript-modal/main.js]: You can remove the `if` statement altogether, since `true` is always `true`.

File: javascript-modal/main.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+$noButton.addEventListener('click', close);
+function close(event) {
+  if (true) {
+    $textBoxContainer.className = 'none';
+    $overlay.className = 'none';
+  }
```
Comment (thebearingedge) [inline javascript-modal/main.js]: You can remove the `if` statement altogether, since `true` is always `true`.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**